src/components/accueil/Accueil.tsx

```diff
@@ -37,6 +38,9 @@ export const Accueil = () => {
     }, {}),
     pelico: true,
   });
+
+  const [activitiesFiltered, setActivitiesFiltered] = React.useState<Activity<AnyData>[]>([]);
```

Comment:
When you have this pattern with a state variable and an effect that updates it:
```typescript
const [myVariable, setMyVariable] = React.useState<...>(...);

React.useEffect(() => {
    [...]
    setMyVariable(...);
}, [props])
```

It's cleaner and easier to use directly a memo:
``` typescript
const myVariable = React.useMemo(() => {
    [...]
    return ...;
}, [props]);
```

Reply:
I made the change and it seems to work well.